presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1050,7 +1052,12 @@ private void printAssignments(NodeRepresentation nodeOutput, Assignments assignm
                     // skip identity assignments
                     continue;
                 }
-                nodeOutput.appendDetailsLine("%s := %s", entry.getKey(), formatter.formatRowExpression(entry.getValue()));
+                if (isExpression(entry.getValue())) {
```

Comment:
We shouldn't expect this to be an `Expression`. Fix the test rather than having it handled here.

Reply:
During transition, we will have tests with both expression and row expression. If test fails, we won't be able to print out the plan which will be very confusing to debug at first. I would prefer we handle both instead of throw an exception. We can remove the logic later once we have moved all tests to row expression.

Reply:
Updated the plan assertion to make sure we always do a translation before print.